lib/podcast_importer.rb

```diff
@@ -0,0 +1,267 @@
+# import script for a podcast live in feeder & dovetail
+require 'prx_access'
+require 'addressable/uri'
+require 'feedjira'
+require 'itunes_category_validator'
+
+class PodcastImporter
```

Comment:
I wrote the `lib/podcast_importer.rb` first, thinking I could do everything with API calls so that the importer could be run anywhere - e.g. you can run it on your local machine agains the prod cms and feeder APIs to import a podcast.

...but it was problematic.
First off - slow to hit the API for everything.
Lots of creates then having to do updates and publish requests after.
Couldn't directly set the published_at date to match the feed.

And basically not useful to integrate with publish.
If we want this to work with publish, needed an API endpoint, and to do the processing async.
But if it is async, you need a model to keep track of how the import is going.

So, I created a podcast import model to both keep track of the progress of the import, and to be accessible via the API to track that progress.

In short, the lib importer is good for importing test data, or other dev tasks, and honestly I spent a long time on it, and I wasn't going to delete it.

The model importer is meant to be integrated into publish so a user could start an import prpcess of their existing podcast.